hack/jenkins/test-image/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM golang:1.5.3
+FROM golang:1.6
```

Comment:
seems to me like CI should be pushing ahead until we find a problem, rather than lagging what all of our users are likely to be doing...